src/mesh/index_derivs.cxx

```diff
@@ -2635,7 +2635,7 @@ const Field2D Mesh::indexFDDX(const Field2D &v, const Field2D &f, CELL_LOC outlo
   if ((method == DIFF_SPLIT) || ((method == DIFF_DEFAULT) && (fFDDX == nullptr))) {
```

Comment:
I know you've not changed this but we only ever pass `method = DIFF_DEFAULT` so why do we allow/perform the option `DIFF_SPLIT` version?

Reply:
I think it's possible for a user to call the `FDDX` function with a `method=DIFF_SPLIT` argument that would be passed through and have an effect here.